src/PlusDataCollection/vtkPlusGenericSerialDevice.cxx

```diff
@@ -246,8 +324,16 @@ PlusStatus vtkPlusGenericSerialDevice::ReceiveResponse(std::string& textReceived
       if (vtkPlusAccurateTimer::GetSystemTime() - startTime > this->MaximumReplyDurationSec)
       {
         // waiting time expired
-        LOG_ERROR("Failed to read complete line from serial device. Received: " << textReceived);
-        return PLUS_FAIL;
+        if (textReceived.empty())

```

Comment:
Make the behavior configurable. Add an option "AcceptReply":
* REQUIRE_LINE_ENDING (default): current behavior
* REQUIRE_NOT_EMPTY: the behavior that you implemented in this pull request
* ANY: response either empty or not empty is returns with success